File: BackEnd/src/main/java/com/example/backend/data/models/Lesson.java
package com.example.backend.data.models;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;

@Getter
@Setter
@Entity
@Table(name = "lessons")
public class Lesson {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    private String name;
    private String teacher;
    private String information;
}

File: TelegramBot/TelegramBot/Data/Entities/Lesson.cs
using Newtonsoft.Json;

namespace TelegramBot.Data.Entities
{
    public class Lesson
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Teacher { get; set; }
        public string Information { get; set; }
    }
}
